contracts/axelar-gas-service/tests/test.rs

```diff
@@ -181,6 +181,91 @@ fn pay_gas_for_contract_call() {
     );
 }
 
+#[test]
+fn fail_add_gas_zero_gas_amount() {
```

Comment:
add unit test for failure if approval wasn't set before calling add_gas

Reply:
updatd.